docs/contributing.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+Open blog is before all an open source project to share a publication of blog posts with the community.
+
+> Developed by the community for everyone freely
+
+## Share your blog posts :rocket:
+
+Before adding your blog post to Open blog, you should check those 3 steps.
+
+### Post rules
+
+- Post should follow the [Code of Conduct](https://github.com/open-blog-initiative/open-blog.dev/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
Add the before post would make sense nop ? 🤔

Reply:
What do you mean ?